pkg/cloud/services/network/routetables.go

```diff
@@ -320,6 +321,13 @@ func (s *Service) getNatGatewayPrivateRoute(natGatewayID string) *ec2.CreateRout
 	}
 }
 
+func (s *Service) getNat64PrivateRoute(natGatewayID string) *ec2.CreateRouteInput {
+	return &ec2.CreateRouteInput{
+		NatGatewayId:             aws.String(natGatewayID),
+		DestinationIpv6CidrBlock: aws.String(services.NAT64CidrBlock),
```

Comment:
I am not sure if we could mark this as optional (e.g.: under `Vpc.Ipv6.EnableNAT64`), then create NAT Gateways (and this role) only when this is true, so we can make ipv6-only clusters without NAT gateways (and it's costs). Is that make sense?

Do we need to allocate one-per-az Nat Gateway (which is a expensive resource)? I wonder if we can create minimum topology (single, or dual, instead per-AZ) of NAT Gateways on IPv6 clusters, helping customers to eliminate public IPv4 address from their environment.

> topology

Something like: `Vpc.Ipv6.NAT64TopologyStrategy: default|single|dual`

Where:
- default: one per az, not changing the default (zonal redundant)
- single: one NGW for entire cluster (SPOF)
- dual: dual NGW with HA in two AZs (if cluster is deployed on 1+ AZs)